finance_manager/src/main/java/com/exadel/finance/manager/user/service/api/UserService.java

```diff
@@ -0,0 +1,17 @@
+package com.exadel.finance.manager.user.service.api;
+
+import com.exadel.finance.manager.user.model.entity.User;
+import com.exadel.finance.manager.user.model.entity.UserModel;
+
+public interface UserService {
+
+    User createUser(UserModel user);
+
+    User readUser();
+
+    User updateUser(UserModel user);
```

Comment:
how would updateUser know which exactly User to update? UserRequestDto will contain ID inside?